app/viewmodels/models/PensionSchemeViewModel.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.models
+
+import play.api.mvc.Call
+import viewmodels.DisplayMessage
+
+case class PensionSchemeViewModel(
```

Comment:
Might be a little clunky creating a view modeland having to pass `DisplayMessage`. Create a new apply method in the companion object with a signature `def apply(title: String, heading: String, onSubmit: Call): PensionSchemeViewModel` that applies the string values to `DisplayMessage`